modules/ROOT/pages/vpn-troubleshooting.adoc

```diff
@@ -7,32 +7,36 @@ These are some of the possible errors you may encounter when attempting to conne
+If you are unable to connect to the Anypoint VPN, ensure any firewalls that are configured are allowing traffic through the IP addresses in the `localExternalIpAddress` field in the VPN tunnels.
 
-=== `NO_PROPOSAL_CHOSEN` Error or Cannot Establish Phase 1 Connection
+=== `NO_PROPOSAL_CHOSEN` Error or `Cannot Establish Phase 1 Connection`
 
-This is likely due to a problem with your Phase 1 configuration. Anypoint VPN supports only IKEv1 connections, so any IKEv2 connections will not work. Additionally, the following Phase 1 Diffie-Hellman (DH) groups are supported: `2 14-18 22 23 24`
+This is likely due to a problem with your Phase 1 configuration.
+Anypoint VPN connections support only IKEv1, meaning IKEv2 do not work.
 
-=== Cannot Establish a Phase 2 Connection
+Phase 1 Diffie-Hellman (DH) groups that are supported include 2, 14-18, 22, 23, and 24. 
 
-It is possible that your Phase 2 DH group is not supported. The supported Phase 2 DH groups are: `2 5 14-18 22 23 24`
+=== Cannot Establish a Phase 2 Connection
 
+It is possible that your Phase 2 DH group is not supported. Phase 2 DH groups that are supported include 2, 5, 14-18, 22, 23, and 24. 
 
-=== Anypoint VPN Tunnel Connection Works, but Routes Are Not Propagated
+=== Anypoint VPN Tunnel Connection Works, but Routes are not Propagated
 
 Ensure that the neighbor IP address for the tunnel is taken from the Local point-to-point IP address in the tunnel’s details.
 
-=== Phase 2 SA Established, but Traffic Not Passing Through Tunnel
+=== Phase 2 SA Established, but Traffic not Passing Through Tunnel
 
-The Anypoint VPN connection supports only one Security Association (SA) pair per tunnel, so any more than one traffic selector per connection results in traffic problems.
+The VPN Connection supports only one Security Association (SA) pair per tunnel, so any more than one traffic selector per connection will cause unexpected results.
+
+To solve this, ensure that only one unique SA is used per VPN tunnel connection. If more than one policy is needed, you must consolidate and filter traffic in your network.
 
-To solve this, ensure that only one unique SA is used per Anypoint VPN tunnel connection. If more than one policy is needed, you must consolidate and filter traffic in your network.
 
 === Anypoint VPN Tunnel Disconnects After Period of Inactivity
 
-IPsec is established by sending interesting traffic (traffic that should be encrypted over the Anypoint VPN connection). If there is no interesting traffic, the tunnel disconnects when the timers expire. This is the expected behavior.
+IPsec is established by sending "interesting traffic" (traffic that should be encrypted over the Anypoint VPN connection). If there is no interesting traffic, the tunnel disconnects. This is the expected behavior, and the timeout value is variable.
```

Comment:
why is the fact that the timeout value is variable connected to "this is the expected behavior" with an "and"? Are we saying if you need the connection to last longer you can adjust the timeout value? Or that the connection times out at different intervals on its own, don't worry about it?